### ISSUE TITLE
Set-Cookie field not displayed

### ISSUE THREAD

**Issue by Daimanu06** [esphen/RESTED#83]
When the server returns a _Set-Cookie_ field within its HTTP header, this field doesn't appear in the response header section of the UI.
However it is present in the Network part of the Firefox development tools.

**Comment by esphen** [comment]
Thanks for the report, I'll look at this after I get 2.0 past review

**Comment by esphen** [comment]
So I looked into this, and this appears to be yet another case of _"it's not a bug, it's a feature"_. The web platform does not allow user agents to read the `Set-Cookie` header, instead preferring user agents to handle it natively. i.e. reading `document.cookie` if the cookie applies to the current domain or sending the cookie transparently to the server.

Try the following code against a server that uses the `Set-Cookie` header to verify.

```js
fetch('http://localhost:3000/')
  .then(res => res.headers.get('Set-Cookie'))
  .then(console.log)
// null
```

See also [the spec](https://fetch.spec.whatwg.org/#forbidden-response-header-name)

To sum up: I would love to, but I can't
